src/sanghwa/memberSet.java

```diff
@@ -0,0 +1,96 @@
+package sanghwa;
+
+import jun.Member;
+
+import java.util.*;
+import java.util.function.Predicate;
+import java.util.stream.Collectors;
+
+public class memberSet {
+    public static HashSet<Member> set = new HashSet<>();
+
+    public boolean addMember(String name, int age, int id) {
+        Member member = new Member(name, age, id);
+        boolean isIn = set.add(member);
+        if (isIn){
+            return true;
+        }
+        return false;
+    }
+    public static int getSize(){
+        return set.size();
+    }
```

Comment:
1. 언뜻 보기에 코드가 바로 읽히지 않아요
메서드 사이에 줄바꿈을 통해서 분리해보는것은 어떨까요?

2. ```static``` 키워드가 붙은 변수, 메서드는 JVM내에서 메서드 영역에서 항상 상주하게 됩니다
메서드 영역에서 상주라 함은 GC(가비지 컬렉션)의 영향을 받지 아니하게 되는데요
이렇게 되면 프로그램 종료시에만 메모리내에서 제거되기에 JVM내부에서 생명주기가 가장 긴 메서드가 됩니다.
메모리 관리에 신경써야할 자바 개발자라면
```static``` 키워드는 서비스의 요구사항에 맞게 적절히 사용되어야합니다.

3. ```return```은 메서드 내부 핵심 로직과의 줄바꿈을 통해서 작성되는것이 관례에요!
지금부터라도 좋은 습관을 만들어봐요!
하지만 메서드 내부 로직 코드가 ```return``` 만 존재한다면 ```getSize()``` 메서드와 같이 줄바꿈을 하지 않아도 됩니다!

```suggestion
    public boolean addMember(String name, int age, int id) {
        Member member = new Member(name, age, id);
        boolean isIn = set.add(member);
        if (isIn){
            return true;
        }
        
        return false;
    }
    
    public static int getSize(){
        return set.size();
    }
    
```

어떤 경우에 ```static```키워드를 사용하면 장점이 작용하고 또한 남발하게 되면 어떤 단점이 생기게 될지 공부해보세요!